acceptance/openstack/loadbalancer/v2/loadbalancers_test.go

```diff
@@ -101,10 +102,18 @@ func TestLoadbalancersCRUD(t *testing.T) {
 	tools.PrintResource(t, newListener)
 
 	// L7 policy
-	_, err = CreateL7Policy(t, lbClient, listener, lb)
+	policy, err := CreateL7Policy(t, lbClient, listener, lb)
 	if err != nil {
 		t.Fatalf("Unable to create l7 policy: %v", err)
 	}
+	defer DeleteL7Policy(t, lbClient, policy.ID)
```

Comment:
This should be

```go
defer DeleteL7Policy(t, lbClient, lb.ID, policy.ID)
```

Reply:
Yes. I wonder why there is no complaint in the CI?

Reply:
Because OpenLab only runs a subset of tests at this time.